### PR TITLE
[Port] Citadel Atmos Adjustments and Relief Valves

### PR DESCRIPTION
**Pull request by SynnGraffkin**
## About The Pull Request

Ports:
- [Adds a couple relief valves #10501](https://github.com/Citadel-Station-13/Citadel-Station-13/pull/10501)
- [Relief valve fixes #10722](https://github.com/Citadel-Station-13/Citadel-Station-13/pull/10722)
- [Adds click interactions to atmos heater/freezer #8800](https://github.com/Citadel-Station-13/Citadel-Station-13/pull/8800) 
- Nerfs TEG down to Citadel Efficiency

## Why It's Good For The Game

Adds relief valves to the game. If the pipe hits a certain pressure, it opens up and releases the gas into the area. Notable example from the PR ported from was them replacing the Atmos Waste injector with it. I figured since I was grabbing anything related to TEGs, this would be a good addition in helping over pressured pipes. **Thanks SuperYodeler for getting TGUI Next working for it.**

Adds Investigation / admin notices for Heaters and Freezers. In an attempt to grab everything related I just grabbed the entire thermo generator file and those lines were the only differences. 

Nerfs TEG efficiency down to a **reasonable** level that is proven to be a decent number to go with based off of Citadel's history. Certain freezer / heater TEG setups are not as strong as before, which is the intended purpose.

Tested and functional. Ran the same tests on a Citadel and Wasp code server and had the same results.

## Changelog
:cl:
add: Citadel's Relief Valves for piping
tweak: Minor tweaks to Heaters / Freezers in regards to admin investigation and notices.
balance: TEG Efficiency lowered
/:cl:

### CODE REVIEW

**Comment by CRITAWAKETS** [comment]
Out of curiosity, how much power does a freezer/heater TEG make with this PR?

**Comment by CRITAWAKETS** [comment]
Also, please do not do a PR with a new feature and an unrelated balance change at the same time. Atomize.

**Comment by dinnerwoerror** [comment]
yea bitch boy, only I'm allowed to do that

**Comment by SynnGraffkin** [comment]
> an unrelated balance change

I made this PR specifically for TEG. I specifically looked at TEG, Thermomachine, and relief valves, and compiled the changes and additions here. The goal was to import any changes to those two machines from Citadel, as well as bring in relief valves from Citadel, in order to try and improve the TEG. Considering the TEG is an atmospheric engine, this is entirely related.

**Comment by Superyodeler** [comment]
While they are related, I do think the relief valves could exist separately to the efficiency change.

**Comment by SynnGraffkin** [comment]
I want Mark's opinion on the matter, because the main reason I even made this PR was because of Crit's overnerfing of the TEG.

**Comment by MarkSuckerberg** [comment]
As much as this should be atomized, I believe it's best to start somewhere. I'll testmerge it so that we can see how efficient it is, and then base our decision on where to go from there on it.

File: tgui/src/interfaces/atmos_relief.ract [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+		<ui-button icon='pencil' action='open_pressure' params='{"open_pressure": "input"}'>Set</ui-button>
+		<ui-button icon='plus' state='{{data.open_pressure == data.max_pressure ? "disabled" : null}}' action='open_pressure' params='{"open_pressure": "max"}'>Max</ui-button>
+		<span>{{Math.round(adata.open_pressure)}} kPa</span>
+	</ui-section>
+	<ui-section label='Close Pressure'>
+		<ui-button icon='pencil' action='close_pressure' params='{"close_pressure": "input"}'>Set</ui-button>
+		<ui-button icon='plus' state='{{data.close_pressure == data.open_pressure ? "disabled" : null}}' action='close_pressure' params='{"close_pressure": "max"}'>Max</ui-button>
+		<span>{{Math.round(adata.close_pressure)}} kPa</span>
+	</ui-section>
+</ui-display>
```
Comment (MarkSuckerberg) [inline tgui/src/interfaces/atmos_relief.ract:12]: Please keep in mind whoever ports TGUI 3.0 will need to update this, the clonepod UI, and the fax machine UI.

Comment (SynnGraffkin) [reply tgui/src/interfaces/atmos_relief.ract:12]: The fuck is TGUI 3.0?

**Comment by MarkSuckerberg** [comment]
magic

**Comment by MarkSuckerberg** [comment]
[TEG Rebalances/Changes](https://trello.com/c/B8tSgqyN/27-teg-rebalances-changes)

**Comment by MarkSuckerberg** [comment]
Merging as it beat out #153 in a vote and I need to update the TGUI to version 3.0 for the latest TG merge.

**Pull request merged**